pyccl/tracers.py

```diff
@@ -447,6 +448,38 @@ def get_angles_derivative(self):
         """
         return np.array([t.der_angles for t in self._trc])
 
+    def get_chi_fft(self, tracer, Nchi, chimin, chimax, ell):
```

Comment:
Just one final query before I approve: could we make this function name a bit more verbose, and also make it a private function (so it does not form part of the API, in case we want to change it in the near future)? Suggestion: `_get_fkem_fft`.

Same query for `_set_fkem_fft`